examples/examples-gpu/nyc-taxi/rf-rapids-dask.ipynb

```diff
@@ -4,9 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>I tend to like to describe the client as the channel for the user to communicate with the cluster- I'm not sure "object that your code uses to do Dask things" is clear enough for a new user. I'm thinking about a better phrasing...</p><p> </p><p>"object that lets you send instructions" </p><p>"object that you use to send commands"</p><p> </p><p>something something?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
That might be sufficient- @rikturr what do you think?

Reply:
I like the explanation of the client being the way you interact with Dask (because thats also used for local cluster Dask stuff). Then the cluster get passed in to there to tell the client to use this particular cluster. It's a nice way to show how its mostly Dask OSS stuff then Saturn just provides the infra layer.

Also I don't think people would get super confused with the fact that "cluster" is for the object and the actual collections of machines, since the object is sorta pointing to that collection of machines.

Reply:
I agree with what you said @rikturr but I'm confused about what it means exactly for this review. Are you ok with the one-liner "tells the cluster to do things"? If not do you have a preferred alternative?